crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -4285,6 +4377,19 @@ impl ConsensusCommitOutput {
             self.active_jwks.into_iter().map(|j| (j, ())),
         )?;
 
+        batch.insert_batch(
+            &tables.congestion_control_object_debts,
+            self.congestion_control_object_debts
+                .into_iter()
+                .map(|(object_id, debt)| (object_id, (self.consensus_round, debt))),
+        )?;
+        batch.insert_batch(
+            &tables.congestion_control_randomness_object_debts,
+            self.congestion_control_randomness_object_debts
+                .into_iter()
+                .map(|(object_id, debt)| (object_id, (self.consensus_round, debt))),
+        )?;
```

Comment:
Question: do we clear objects whose debt are 0? This needs to be done in the same batch here.